reana_job_controller/cern_htcondor_job_manager.py

```diff
@@ -0,0 +1,218 @@
+            'cp /job_wrapper.sh {};'.format(self.workflow_workspace)
+        chown_worksapce_cmd = \
+            'sudo chown -R $CONDOR_USER {};'.format(self.workflow_workspace)
+        submission_cmd = \
+            ('job_id=$(condor_submit -spool -terse {} | '
+             'cut -f 1 -d " " );'.format(submission_file))
+        job_status_cmd = \
+            'job_status="eval condor_q $job_id -format %d JobStatus";'
+        job_tracking_cmd = \
+            ('while [ $($job_status) -ne 4 ];'
+             ' do echo "JOB $job_id is still running"; sleep 30; done;')
+        add_user_cmd = 'useradd -Ms /bin/bash $CONDOR_USER;'
+        get_output_cmd = 'condor_transfer_data $job_id;'
+        cmd = \
+            ["{}{}{} sudo -u $CONDOR_USER /bin/bash -c '{}{}{}{}{}{}'".format(
+                add_user_cmd,
+                cp_job_wrapper_cmd,
+                chown_worksapce_cmd,
+                go_to_workspace_cmd,
+                authentication_cmd,
+                submission_cmd,
+                job_status_cmd,
+                job_tracking_cmd,
+                get_output_cmd
+            )]
+        submission_job = self._create_job_spec(
+            command=cmd,
+            env_vars=[{'name': 'CONDOR_USER',
+                      'value': CERN_USER}])
+        try:
+            api_response = \
+                current_k8s_batchv1_api_client.create_namespaced_job(
+                    namespace=K8S_DEFAULT_NAMESPACE,
+                    body=submission_job)
+            return api_response.metadata.labels['job-name']
+        except ApiException as e:
+            logging.debug("Error while connecting to Kubernetes"
+                          " API: {}".format(e))
+        except Exception as e:
+            logging.error(traceback.format_exc())
+            logging.debug("Unexpected error: {}".format(e))
+
+    def _create_job_spec(self, name=None, command=None, image=None,
+                         env_vars=None):
+        """Create a spec of kubernetes job for condor job submission.
+
+        :param name: Name of the job.
+        :type name: str
+        :param image: Docker image to use for condor job submission.
+        :type image: str
+        :param command: List of commands to run on the given job.
+        :type command: list
+        :param env_vars: List of environment variables (dictionaries) to
+            set on execution container.
+        :type list: list
+        """
+        name = name or 'htcondor-submitter-' + str(uuid.uuid4())
+        image = image or current_app.config['HTCONDOR_DOCKER_IMG']
+        command = command or []
+        env_vars = env_vars or []
+        if isinstance(command, str):
+            command = [command]
+        elif not isinstance(command, list):
+            raise ValueError('Command should be a list or a string and not {}'
+                             .format(type(command)))
+
+        workflow_metadata = client.V1ObjectMeta(name=name)
+        job = client.V1Job()
```

Comment:
If we have time now we should abstract this creation, we are repeating it here and in R-Workflow-Controller, see [here](https://github.com/reanahub/reana-workflow-controller/blob/2709032255454bc0c9be85067d2b29eb75c1efe2/reana_workflow_controller/workflow_run_manager.py#L315). Otherwise, let's merge and create an issue.